frontend/src/mixins/getSecureExchangeContactMixin.js

```diff
@@ -0,0 +1,43 @@
+//this mixin has functions to use the Secure Exchange object contact type code
+//to render the correct contact name to display based on contact type code.
+//whether it is a School or Minteam.
+//ie SCHOOL -> take the contact identifier (UUID) to check the list of schools for the correct name
+
+import {mapState} from 'vuex';
+
+export default {
+  mounted() {
+    this.$store.dispatch('app/getCodes');
+    this.$store.dispatch('edx/getCodes');
+  },
+  computed: {
+    ...mapState('edx', ['ministryTeams']),
+    ...mapState('app', ['mincodeSchoolNames']),
+  },
+  methods: {
+    getContactName(secureExchange) {
+      let contactName = '';
+
+      switch (secureExchange.secureExchangeContactTypeCode) {
+      case 'MINTEAM' :
+        if (this.ministryTeams.length > 0) {
+          let ministryTeam = this.ministryTeams.find((minTeam) => minTeam.ministryOwnershipTeamId === secureExchange.ministryOwnershipTeamID);
+          contactName = ministryTeam?.teamName || 'minteam not found';
```

Comment:
Since it will be visible, please qualify it "Ministry team was not found"

Reply:
changed as well